modules/thermo-cycler/thermo-cycler-arduino/lid.cpp

```diff
@@ -246,11 +240,14 @@ bool Lid::open_cover()
      return true;
   }
   motor_on();
-  move_angle(-1);    // move down a bit
-  solenoid_on();
-  delay(100);
-  move_angle(5);     // move up a bit
-  solenoid_off();
+  if(move_angle(-5))  // move down a bit
```

Comment:
Just to be absolutely sure, should we move `-LID_MOTOR_RANGE_DEG` here?

Reply:
Good call. Hardware also requested to abort lid open if bottom switch is not pressed after that step. I'll make those changes